src/test/java/org/broadinstitute/hellbender/tools/variantdb/nextgen/ExtractCohortRecordUnitTest.java

```diff
@@ -0,0 +1,138 @@
+package org.broadinstitute.hellbender.tools.variantdb.nextgen;
+
+import org.apache.avro.generic.GenericRecord;
+import org.broadinstitute.hellbender.GATKBaseTest;
+
+import org.broadinstitute.hellbender.utils.bigquery.AvroFileReader;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class ExtractCohortRecordUnitTest extends GATKBaseTest {
+
+    private static GenericRecord getSingleRecordFromAvroFile (String inputFileName) {
+        final AvroFileReader avroFileReader = new AvroFileReader(inputFileName);
+        return avroFileReader.next();
+    }
+
+    // load in test data
+    private static final String testFileDir = "src/test/resources/org/broadinstitute/hellbender/tools/variantdb/nextgen/ExtractCohortRecord/";
+    private static final GenericRecord definedInputGenericRecord = getSingleRecordFromAvroFile(testFileDir + "test_input_defined.avro");
+    private static final GenericRecord nulledInputGenericRecord = getSingleRecordFromAvroFile(testFileDir + "test_input_nulls.avro");
+
+    // generate records to test
+    private ExtractCohortRecord allDefinedRecord = new ExtractCohortRecord(definedInputGenericRecord);
+    private ExtractCohortRecord someNullsRecord = new ExtractCohortRecord(nulledInputGenericRecord);
+
+    @Test
+    public void testGetContig() {
```

Comment:
just a style thing... but I might have written this as just two tests.  `testAllFields` and `testNullableFields` or something and then just do all the asserts.

Reply:
that's way better, thank you